examples/tests/valid/ulid.test.w

```diff
@@ -0,0 +1,35 @@
+bring util;
+bring expect;
+
+inflight class Sortable {
+  pub static extern "./sort.js" sortStringListAsc(arr: Array<str>): Array<str>;
+  pub static extern "./sort.js" sortStringListDesc(arr: Array<str>): Array<str>;
+}
+
```

Comment:
Makes sense to test it in preflight as well, right?

Reply:
just tried it, but ran into issues with other tooling to force a sleep. so just added a simple preflight test.

- https://github.com/winglang/wing/pull/5417
- https://github.com/winglang/wing/issues/5418